### ISSUE TITLE
Doesn't work with Node 5.x.

### ISSUE THREAD

**Issue by genu** [FountainJS/generator-fountain-webapp#115]
There error is:

```
TypeError: this.options.ci.includes is not a function
    at writing (/usr/local/lib/node_modules/generator-fountain-webapp/node_modules/generator-fountain-gulp/generators/ci/index.js:8:25)
```

This is probably due to: See https://github.com/nodejs/node/issues/5715 indicating that the `includes` is currently hidden behind a harmony flag

**Comment by zckrs** [comment]
You use a outdated Node v5 IMO

**Comment by zckrs** [comment]
Can you paste your `.yo-rc.json` ?

And execute in your shell

```
node -e "var os=require('os');console.log('Node.js ' + process.version + '\n' + os.platform() + ' ' + os.release())"
yo --version
npm --version
```

**Comment by genu** [comment]
Updated node to latest 6.x and it works now. Thanks! I suppose I didn't bother updating node since the instructions indicate that at least 4.x is required (I was on 5.7.1).

**Comment by micaelmbagira** [comment]
Yes, it's an issue, `includes` is not compatible with Node 4

**Comment by genu** [comment]
We can just close this issue and make a note in the docs on the node version requirement.

**Comment by elartix** [comment]
Same situation, I use node 4.4.7 and npm 3.10.5

```
j:\Projects\Elartix\fwapp\p2>node -v
v4.4.7
```

```
j:\Projects\Elartix\fwapp\p2>npm -v
3.10.5
```

```
j:\Projects\Elartix\fwapp\p2>npm -g view generator-fountain-angular1 version
0.7.0
```

```
j:\Projects\Elartix\fwapp\p2>yo fountain-angular1
? Which module management do you want? None with Bower and script injection
? Which JS preprocessor do you want? Pure old JavaScript
? Which CSS preprocessor do you want? Less
? Which Continuous Integration platform do you want? (Press <space> to select)
? Do you want a sample app? A working landing page
? Would you like a router? Angular UI Router
? May fountainjs.io anonymously report usage statistics to improve the tool over time? No
events.js:141
      throw er; // Unhandled 'error' event
      ^

TypeError: this.options.ci.includes is not a function
    at writing (C:\Program Files\nodejs\node_modules\generator-fountain-angular1\node_modules\generator-fountain-gulp\generators\ci\index.js:7:25)
    at Object.<anonymous> (C:\Program Files\nodejs\node_modules\generator-fountain-angular1\node_modules\yeoman-generator\lib\base.js:431:23)
    at C:\Program Files\nodejs\node_modules\generator-fountain-angular1\node_modules\run-async\index.js:26:25
    at C:\Program Files\nodejs\node_modules\generator-fountain-angular1\node_modules\run-async\index.js:25:19
    at C:\Program Files\nodejs\node_modules\generator-fountain-angular1\node_modules\yeoman-generator\lib\base.js:432:9
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by zckrs** [comment]
Just released v0.7.1 who fixed your issue
